### ISSUE TITLE
Wrong screen reader announcement in quiz levels

### ISSUE THREAD

**Issue by dnikub** [a11yphant/a11yphant#96]
### Reported by @micmath

Hi,

When I take a quiz using Apple VoiceOver screen reader (in Chrome browser) latest versions, I do not hear the correct content announced. As you can see from the attached screen capture, the visually displayed content is "Wrong answer, try again!" but the screen reader announces "The submission is being processed." There is no additional announcement by VoiceOver so a person who cannot see the screen cannot perceive if their answer is wrong.

The url is [https://a11yphant.com/challenge/a-valid-html-document/level/02](https://a11yphant.com/challenge/a-valid-html-document/level/02)

You can fix this by adding `role="alert"` (or `role="status"` if you want to be less assertive) to the HTML element that has the results text. Like this:

`<div role="alert" class="col-span-3 mb-8 p-1 overflow-y-auto">...</div>`

See [https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA22](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA22)

Thank you,
Michael
